lighthouse-core/report/html/templates.html

```diff
@@ -18,7 +18,7 @@
 <!-- Lighthouse run warnings -->
 <template id="tmpl-lh-warnings--toplevel">
   <div class="lh-warnings lh-warnings--toplevel">
-    <strong class="lh-warnings__msg"></strong>
+    <p class="lh-warnings__msg"></p>
```

Comment:
@hwikyounglee just wanna double check we want to make this be not bold.


this:
![image](https://user-images.githubusercontent.com/4071474/81484320-7a1fa580-91f9-11ea-94b0-63bac26c018e.png)
![image](https://user-images.githubusercontent.com/4071474/81484345-a89d8080-91f9-11ea-92b2-905da2dd2524.png)


or that:
![image](https://user-images.githubusercontent.com/4071474/81484316-7429c480-91f9-11ea-816e-2955c46905cb.png)
![image](https://user-images.githubusercontent.com/4071474/81484340-a2a79f80-91f9-11ea-843b-df29b6018f38.png)

Reply:
Per the design

- No bold
- On the light mode: the title 'There were...' should be in orange #d04900

Could we try the color for the light mode? 
I think the colors would do it without bold weight. 

Side note: the orange in dark mode is #ffa400

Reply:
the color contrast for those light colors #d04900 (text) and rgb(255, 246, 234) (background) has too low contrast according to axe :(

Reply:
Wondering if we're in a place to add or replace the color with #BD4200 

It's 4.97 
https://contrast-ratio.com/#%23BD4200-on-%20rgb%28255%2C%20246%2C%20234%29

Reply:
![image](https://user-images.githubusercontent.com/4071474/81749142-ee5b8280-945f-11ea-8a8c-336f7aa067cf.png)

seems good to me.